src/business.ts

```diff
@@ -299,7 +299,7 @@ export class BusinessApi {
     sortBy?: string,
   } = {}): Promise<{
     success: boolean,
-    body?: TransitionList,
+    business?: TransitionList,
```

Comment:
This is a list of Transitions... so I'm pretty sure I would not call it `business`...